### PR TITLE
Fix documentation and math display errors

### PR DESCRIPTION
**Pull request by laurensvalk**
- This changes 

   Screenshot from 2020-08-14 14-05-35

   to:

   image

- This changes:

   Screenshot from 2020-08-14 14-13-53

   to

   Screenshot from 2020-08-14 14-14-30

   Inverse has now been changed to `^{-1}` as well.

- This changes:

    Screenshot from 2020-08-14 14-09-21

   to:

   image

- and a few other minor fixes.

### CODE REVIEW

File: control/timeresp.py [hunk 1]
```diff
@@ -249,7 +249,7 @@ def forced_response(sys, T=None, U=0., X0=0., transpose=False,
     Notes
     -----
     For discrete time systems, the input/output response is computed using the
-    :scipy-signal:ref:`scipy.signal.dlsim` function.
+    scipy.signal.dlsim function.
```
Comment (bnavigator) [inline control/timeresp.py:252]: Just fix the broken intersphinx mapping (untested):

```suggestion
    :scipy:ref:`scipy.signal.dlsim` function.
```

https://github.com/python-control/python-control/blob/ec8cbf053a8dee367e925f54eb42aa256a8d3265/doc/conf.py#L99-L101

**Comment by bnavigator** [comment]
> * This changes:
>   Screenshot from 2020-08-14 14-09-21
>   to:
>   Screenshot from 2020-08-14 14-06-19

I think your "fixed" screenshot shows the wrong output portion.

File: control/timeresp.py (hunk 1 above)
Comment (laurensvalk) [reply control/timeresp.py]: Thanks! I've updated the commits to fix it.

**Comment by laurensvalk** [comment]
Thanks for the review. I've adapted it based on your feedback and fixed the screenshot above.

**Comment by bnavigator** [comment]
I submitted a PR into your branch to take this even further: pybricks#1

**Comment by laurensvalk** [comment]
Sounds good - I just merged those into this PR.

**Pull request merged**